pkgs/by-name/yt/ytmdesktop/package.nix

```diff
@@ -0,0 +1,97 @@
+{
+  lib,
+  asar,
+  binutils,
+  commandLineArgs ? "",
```

Comment:
```suggestion
```
Rebuilding the derivation just to change that, is kinda wasteful

Reply:
This is useful for setting values like `--password-store="gnome-libsecret"`, which is sometimes necessary to overcome [safeStorage detection outside of predefined desktop environments](https://github.com/electron/electron/issues/39789). The only other way I've thought of setting a default CLI flag would be to override the whole `makeDesktopItem`.

I've seen [other chromium based packages do the same](https://github.com/search?q=repo%3ANixOS%2Fnixpkgs%20--add-flags%20%24%7Blib.escapeShellArg%20commandLineArgs%7D&type=code), but I'm open to alternatives.